src/App.tsx

```diff
@@ -3,62 +3,15 @@ import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import classNames from 'classnames';
-import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
-import { Loader } from './components/Loader';
+import { GlobalProvider } from './components/GeneralContext';
+import { MainContent } from './components/MainContent';
 
-export const App: React.FC = () => {
-  return (
+export const App: React.FC = () => (
+  <GlobalProvider>
     <main className="section">
       <div className="container">
```

Comment:
Would be logical if `main` tag existed inside `MainContent` component - move these 2 wrappers there